core/src/storage/__init__.py

```diff
@@ -1,21 +1,27 @@
-from storage import cache, common, device
-from trezor import config
-
-
-def set_current_version() -> None:
-    device.set_version(common.STORAGE_VERSION_CURRENT)
-
-
 def wipe() -> None:
+    from trezor import config
+    from storage import cache
+
     config.wipe()
     cache.clear_all()
 
 
 def init_unlocked() -> None:
+    from storage import common, device
+
     # Check for storage version upgrade.
     version = device.get_version()
     if version == common.STORAGE_VERSION_01:
-        _migrate_from_version_01()
+        # _migrate_from_version_01
```

Comment:
please move this back to a separate function, the clarity in this part is more important than the saving

Reply:
Done in [6ef97fa](https://github.com/trezor/trezor-firmware/pull/2525/commits/6ef97fabaa52631d48ea7387eaa1b666850c054d)